.github/workflows/NodeJS.yml

```diff
@@ -29,7 +29,7 @@ env:
 jobs:
   linux-nodejs:
     name: node.js Linux
-    runs-on: ubuntu-20.04
```

Comment:
I think this would impact the minimum glibc version requirement for the node binaries, no?